torch_geometric/nn/conv/message_passing.py

```diff
@@ -353,6 +356,59 @@ def propagate(self, edge_index: Adj, size: Size = None, **kwargs):
 
         return out
 
+    def edge_update(self, edge_index: Adj, size: Size = None, **kwargs):
```

Comment:
I think `size` can be dropped, as `size` is only useful for message aggregation.

Reply:
makes sense I'll update